### ISSUE TITLE
how validate csv file only

### ISSUE THREAD

**Issue by jayanty3** [Bunlong/react-papaparse#37]

**Comment by Bunlong** [comment]
@jayanty3 I don't get your point. Could you pls give me more detail? Thanks!

**Comment by jacobedawson** [comment]
I think what @jayanty3 is asking is: is there a way to ensure that users only upload .csv files. I would also like to know this - we don't seem to have access to the file extension in order to ignore any non-csv files.

**Comment by bobbyrinaldy** [comment]
@jacobedawson you can use File Type while you using onFileLoaded function and just give it validation like usual.

for example :
```js
onFileLoaded={(data, fileInfo) => console.dir(data, fileInfo)} //File info has some filename or type that you can use
```

**Comment by jacobedawson** [comment]
Thanks @bobbyrinaldy, much appreciated :)

**Comment by chrys-exaucet** [comment]
Any progress so far?

**Comment by Bunlong** [comment]
@exaucae Could you try bobbyrinaldy solution?

**Comment by exaucae** [comment]
Yes, I did.  It means we have to enforce file type by writing custom logic. It would have been nice to have it built in.

**Comment by alex-airbox** [comment]
Is this still valid? Issue is open but comment is from 3 years ago. And I can't find that `onFileLoaded` on the `onFileLoaded` definition, just `onUploadAccepted` and `onUploadRejected`. 

And I noticed there's reference to a `accept` property with a sidenote on the docs about it, so maybe this should be closed as it's built in.
